### PR TITLE
Add Identifiable to README

### LINKED ISSUE #3: Add Identifiable protocol to README

### PR DESCRIPTION
**Pull request by 3lvis**
Fixes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,20 @@
...
+Simple protocol extensions to simplify registering a cell in UITableViewController and UICollectionViewController.
+
+**Before**: 
+```
+self.tableView.registerClass(ItemCell.self, forCellReuseIdentifier: "ItemCell")
+```
+
+**After**: 
+```
+self.tableView.register(ItemCell)
```
Comment (elland) [inline README.md:18]: I think it requires `ItemCell.self`, no?

Comment (3lvis) [reply README.md:18]: Nope!

Comment (elland) [reply README.md:18]: 😭 so happy

**Comment by 3lvis** [comment]
Ok, gonna merge this now. Let me know if I should add something else, I can probably do it in a different PR :)

**Pull request merged**